exercises/random/common_functions.py

```diff
@@ -0,0 +1,28 @@
+# Ask for user input and ensures the user enters an integer
+def get_user_input(message):
+    while True:
+        user_input = input(message)
+        if not user_input.isdigit():
+            print("Oops! That was not a valid number. Try again...")
+        else:
+            return int(user_input)
```

Comment:
If you use short-circuit logic (like return) one advantage is *not* having to fill out the alternative branches.

After a return statement you know that the code must be executing in the context of the other boolean condition.

Reply:
I see - I rewrote this inverting the logic and removing the else